sdk/core/azure-core/vcpkg/Config.cmake.in

```diff
@@ -11,7 +11,9 @@ if(@BUILD_TRANSPORT_CURL@)
 endif()
 
 if(WIN32)
-  find_dependency(wil)
+  if (NOT "${CMAKE_SYSTEM_NAME}" STREQUAL "WindowsStore")
```

Comment:
`wil` is only needed by WinHTTP, which is only available on non-UWP WIndows (see full `vcpkg.json` file below).